src/explorer/initFileLoad.js

```diff
@@ -5,80 +5,74 @@ export function initFileLoad() {
   var explorer = this;
   console.log(this);
   explorer.dataFileLoad = {};
-  explorer.dataFileLoad.wrap = explorer.codebook.fileListing.wrap
-    .insert('div', '*')
-    .attr('class', 'dataLoader')
-    .style('margin', 0);
-  explorer.dataFileLoad.top = explorer.dataFileLoad.wrap.append('div');
-  explorer.dataFileLoad.bottom = explorer.dataFileLoad.wrap.append('div');
+  explorer.dataFileLoad.wrap = explorer.codebook.instructions.wrap
+    .append('span')
+    .attr('class', 'dataLoader');
 
-  explorer.dataFileLoad.top
-    .append('small')
-    .text('Add a local .csv file:')
-    .append('sup')
-    .html('&#9432;')
-    .property(
-      'title',
-      'Create a codebook for a local file. File is added to the data set list, and is only available for a single session and is not saved.'
-    )
-    .style('cursor', 'help');
+  explorer.dataFileLoad.wrap.append('span').text(' Load a csv: ');
 
-  var loadStatus = explorer.dataFileLoad.top
-    .append('small')
-    .attr('class', 'loadStatus')
-    .style('float', 'right')
-    .text('Select a csv to load');
+  explorer.dataFileLoad.loader_wrap = explorer.dataFileLoad.wrap
+    .append('label')
+    .attr('class', 'file-load-label');
+
+  explorer.dataFileLoad.loader_label = explorer.dataFileLoad.loader_wrap
+    .append('span')
+    .text('Choose a File');
 
-  explorer.dataFileLoad.loader = explorer.dataFileLoad.bottom
+  explorer.dataFileLoad.loader_input = explorer.dataFileLoad.loader_wrap
     .append('input')
     .attr('type', 'file')
     .attr('class', 'file-load-input')
     .on('change', function() {
+      var files = this.files;
+      explorer.dataFileLoad.loader_label.text(files[0].name);
+
       if (this.value.slice(-4).toLowerCase() == '.csv') {
-        loadStatus
-          .text(this.files[0].name + ' ready to load')
-          .style('color', 'green');
-        explorer.dataFileLoad.dataFileLoadButton.attr('disabled', null);
-      } else {
-        loadStatus
-          .text(this.files[0].name + ' is not a csv')
-          .style('color', 'red');
-        explorer.dataFileLoad.dataFileLoadButton.attr('disabled', true);
-      }
-    });
+        loadStatus.text(' loading ...').style('color', 'green');
+        var fr = new FileReader();
+        fr.onload = function(e) {
+          // get the current date/time
+          var d = new Date();
+          var n = d3.time.format('%X')(d);
 
-  explorer.dataFileLoad.dataFileLoadButton = explorer.dataFileLoad.bottom
-    .append('button')
-    .text('Load File')
-    .attr('class', 'file-load-button')
-    .attr('disabled', true)
-    .style('float', 'right')
-    .on('click', function(d) {
-      //credit to https://jsfiddle.net/Ln37kqc0/
+          addFile.call(explorer, files[0].name, e.target.result);
 
-      var files = explorer.dataFileLoad.loader.node().files;
+          //clear the file input
+          loadStatus.text('Loaded.').style('color', 'green');
```

Comment:
should update this to say _Loaded <csv>._ instead of replacing the _Choose a File_ text.  Captured in #303 .